src/ParallelRun.jl

```diff
@@ -377,33 +412,56 @@ save_results(outdir, box, results) =
 """
 called from main entry point.
 """
-function infer_box(box::BoundingBox, stagedir::String, outdir::String;
-                   timing=InferTiming())
-    Log.debug("multithreaded parallelism only")
+function infer_box(box::BoundingBox, stagedir::String, outdir::String)
+    times = InferTiming()
+
+    Log.info("processing box $(box.ramin), $(box.ramax), $(box.decmin), $(box.decmax) with $(nthreads()) threads")
 
     tic()
     # Get vector of (run, camcol, field) triplets overlapping this patch
     rcfs = get_overlapping_fields(box, stagedir)
-    timing.query_fids = toq()
+    times.query_fids = toq()
 
-    results = one_node_infer(rcfs, stagedir; box=box, timing=timing)
+    catalog, target_sources, neighbor_map, images =
+        infer_init(rcfs,
+                   stagedir;
+                   box=box,
+                   primary_initialization=true,
+                   timing=times)
+
+    #@time results = one_node_single_infer(catalog, target_sources, neighbor_map, images; timing=times)
+    @time results = one_node_joint_infer(catalog, target_sources, neighbor_map, images; timing=times)
 
     tic()
     save_results(outdir, box, results)
-    timing.write_results = toq()
+    times.write_results = toq()
 
-    puts_timing(timing)
+    puts_timing(times)
```

Comment:
These instance of `InferTiming` are called `times` some places and `timing` elsewhere. Could we stick to just one of them?